web-common/src/features/dashboards/time-dimension-details/TDDTable.svelte

```diff
@@ -115,8 +115,8 @@
 
   // Any time visible line list changes, redraw the table
   $: {
-    scrubPos;
-    highlightedCol;
+    // scrubPos;
+    // highlightedCol;
```

Comment:
This removes the hover effect on the column in TDD view when a chart is hovered over.